integrationtests/src/test/java/org/axonframework/integrationtests/deadline/jobrunr/DeadlineDetailsSerializationTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2010-2022. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.integrationtests.deadline.jobrunr;
+
+import org.axonframework.deadline.jobrunnr.DeadlineDetails;
+import org.axonframework.messaging.ScopeDescriptor;
+import org.axonframework.modelling.command.AggregateScopeDescriptor;
+import org.axonframework.modelling.saga.SagaScopeDescriptor;
+import org.jobrunr.utils.mapper.jackson.JacksonJsonMapper;
+import org.junit.jupiter.api.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DeadlineDetailsSerializationTest {
```

Comment:
I don't fully comprehend why there are two  `DeadlineDetailsSerializationTest` test classes... Why aren't these tests in the other one, or vice versa?

Reply:
One is using the classes from modeling, the other isn't. However, after the serialization changes, just having the one in messaging might be enough.

Reply:
Alright, good to know. Nudge me once you've got all that sorted out!

Reply:
@smcvb just the one within the messaging package is kept. It has changed quite a bit through.

Reply:
Gotcha, thanks for the heads-up.